lib/src/scenario/TestScenarioFactory.ts

```
@@ -0,0 +1,600 @@
+      }
+      if (spaceConfig.settings.membership) {
+        await updateSpaceSettings(spaceModel.id, {
+          membership: {
+            policy: spaceConfig.settings.membership.policy,
+          },
+        });
+      }
+    }
+    if (spaceCommunityConfig) {
+      if (spaceCommunityConfig.members) {
+        this.assignUsersByTypeToRole(
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)